entrypoint.sh

```diff
@@ -37,24 +36,40 @@ fi
 
 # Deploy the Fly app, creating it first if needed.
 if ! flyctl status --app "$app"; then
+  # Backup the original config file since 'flyctl launch' messes up the [build.args] section
+  cp "$config" "$config.bak"
```

Comment:
I don't think we want to copy and restore the config. What is you reason or goal with that? 

I'm working on the approach of modifying `flyctl launch` to make more limited changes to the config file under this circumstance.

Reply:
As stated in the comment `flyctl launch` messes up the `[build.args]` section